packages/dialtone-vue2/components/tooltip/tooltip.test.js

```diff
@@ -2,7 +2,6 @@ import { createLocalVue, mount } from '@vue/test-utils';
 import DtTooltip from './tooltip.vue';
 import {
   TOOLTIP_KIND_MODIFIERS,
-  TOOLTIP_DIRECTIONS,
```

Comment:
Whats happen with `TOOLTIP_DIRECTIONS`?

Reply:
<img width="1198" alt="image" src="https://github.com/dialpad/dialtone/assets/89984179/dd08cc17-e7f6-479a-84df-bf11de15a909">

I know sometimes test this can be tricky but if we could reach it we can get the

Reply:
hmm yeah think we could do that.